packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -151,3 +151,39 @@ export class Intc extends Op {
     stack.push(intc);
   }
 }
+
+// pop a value from the stack and store to scratch space
+export class Store extends Op {
+  readonly index: number;
+  readonly interpreter: Interpreter;
+
+  constructor (index: number, interpreter: Interpreter) { // eslint-disable-line sonarjs/no-identical-functions
```

Comment:
you can put the eslint comment to in the top of the file:
```
// eslint-disable sonarjs/no-identical-functions
```

Reply:
done.